src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -136,6 +142,9 @@ public static class DnnCatalog
                 Epoch = epoch,
                 LearningRate = learningRate,
                 BatchSize = batchSize,
+                EnableEarlyStopping = enableEarlyStopping,
+                MinDelta = earlyStoppingminDelta,
+                Patience = earlyStoppingPatience,
```

Comment:
Since you are taking this technique from [https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/callbacks.py#L1143-L1260](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/callbacks.py#L1143-L1260) 

were you also planning to add the "modes", i.e min, max, auto?